am-lib/src/main/java/uk/gov/hmcts/reform/amlib/db/migration/V3__Create_default_roles_config_tables.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TYPE securityclassification AS ENUM ('Public', 'Private', 'Restricted');
+
+CREATE TABLE roles (
+  role_name varchar(100) PRIMARY KEY,
```

Comment:
cosmetic sometimes you write sql keywords lower and sometimes upper case, be consistent... :)

Reply:
all types were lowercase :)

Reply:
ok, fixed. Always uppercase